src/Vendoring/OpenTelemetry.Instrumentation.SqlClient/GlobalSuppressions.cs

```diff
@@ -0,0 +1,11 @@
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE0028:Simplify collection initialization", Justification = "Vendor code", Scope = "namespaceanddescendants", Target = "~N:OpenTelemetry.Instrumentation.SqlClient")]
```

Comment:
Can we just use .editorconfig for this?

Reply:
These are specific to the files from OTel, but I will check if we can set rules for specific paths.

Reply:
Done